codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/transformers/AttachValidationExceptionToConstrainedOperationInputsInAllowList.kt

```diff
@@ -8,14 +8,41 @@ package software.amazon.smithy.rust.codegen.server.smithy.transformers
 import software.amazon.smithy.model.Model
 import software.amazon.smithy.model.shapes.EnumShape
 import software.amazon.smithy.model.shapes.OperationShape
+import software.amazon.smithy.model.shapes.ServiceShape
```

Comment:
We should rename this file, since now it contains more logic than just adding the shape if the service is in the allow list.

Perhaps just `AttachValidationExceptionToConstrainedOperationInputs`.kt?